app/views/admin/organizations/show.html.erb

```diff
@@ -1,61 +1,115 @@
-<header class="flex items-center mb-6">
-  <div>
-    <h1 class="crayons-title"><%= @organization.name %></h1>
-    <p class="color-base-60">Created <%= @organization.created_at.strftime("%b %e '%y") %></p>
-  </div>
-
-  <div class="ml-auto">
-    <%= link_to "View @#{@organization.name}", "/#{@organization.slug}", class: "crayons-btn", target: "_blank", rel: "noopener" %>
-  </div>
+<%= javascript_packs_with_chunks_tag "admin/organizations", defer: true %>
+<header>
+  <section class="crayons-card flex flex-col l:flex-row gap-2 m:gap-4 l:gap-6 mb-2 m:mb-3 p-3 s:p-4 m:p-7">
+    <span class="crayons-logo crayons-logo--3xl">
+      <img src="<%= @organization.profile_image %>" width="100%" height="100%" alt="<%= @organization.name %> profile picture">
+    </span>
+    <div class="grid gap-2 flex-1">
+      <div class="s:flex items-center gap-5">
+        <div class="flex flex-1 items-center">
+          <h1 class="crayons-title lh-tight">
+            <%= @organization.name %>
+          </h1>
+        </div>
+        <div class="flex relative justify-between s:justify-end gap-2 my-2 s:my-0">
+          <%= link_to t("views.admin.organizations.profile.visit"), "/#{@organization.slug}", class: "c-cta", target: "_blank", rel: "noopener" %>
+          <div class="dropdown-trigger-container">
+            <button type="button" class="c-btn c-btn--icon-alone dropdown-trigger" id="options-dropdown-trigger" aria-haspopup="true" aria-expanded="false" aria-controls="options-dropdown">
+              <%= crayons_icon_tag("overflow-vertical", title: t("views.admin.users.profile.options.icon")) %>
+            </button>
+            <div class="crayons-dropdown right-0 left-0 s:left-auto" id="options-dropdown">
+              <ul class="p-0">
+                <li>
+                  <button
+                    type="button" class="c-btn c-btn--destructive w-100 align-left"
+                    data-modal-title="<%= t("views.admin.organizations.delete.heading", organization: @organization.name) %>"
+                    data-modal-size="small"
+                    data-modal-content-selector="#delete-organization">
+                    <%= t("views.admin.organizations.options.delete") %>
+                  </button>
+                </li>
+              </ul>
+            </div>
```

Comment:
I'm not sure what it is, because the styling seems fine, but something about the dropdown feels a bit off to me. It seems big compared to the other elements? This is mostly a nitpick though

Reply:
Thanks, I'll double check this - it should follow the styling from the member manager page.

Reply:
I double checked the layout and it follows the same styles as the member manager page, however because there are less elements in the dropdown so maybe thats why it looks a bit sparse. I then double checked with Anuj to see if we wanted to change it any way and he said its fine from a design perspective 👍